manifests/c/ChilliBits/Spice/0.7.1/ChilliBits.Spice.locale.de-DE.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: ChilliBits.Spice
+PackageVersion: 0.7.1
+PackageLocale: de-DE
+Publisher: ChilliBits
+PublisherUrl: https://chillibits.com/
+PublisherSupportUrl: https://www.spicelang.com
+PrivacyUrl: https://www.chillibits.com/en/legal-notice/
+Author: Marc Auberer
+PackageName: Spice Programming Language
```

```suggestion
PackageName: Spice
```
This must match Control Panel.